src/test/java/ru/testtask/service/ProjectServiceTest.java

```diff
@@ -57,6 +68,7 @@ public void createTest(){
 
     @Test(expected = NameAlreadyExistsException.class)
     public void checkSimilarNamesCreationTest(){
```

Comment:
You need to remove all projects before, because the test will be failed if project already exist